### PR TITLE
update README.md so everything is rendered correctly

### PR DESCRIPTION
**Pull request by Cosaquee**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -422,12 +422,12 @@ There are some zabbix specific parameters, please check them by opening the mani
...
 * `script_dir`: When `script` is used, this parameter can provide the directly where this script needs to be placed. Default: '/usr/bin'
 
-###Reference zabbix-template
+### Reference zabbix-template
 
 * `templ_name`: The name of the template. This name will be found in the Web interface.
 * `templ_source`: The location of the XML file wich needs to be imported.
 
-##limitations
+## limitations
```
Comment (bastelfreak) [inline README.md]: should we switch to "Limitations"?

**Comment by bastelfreak** [comment]
Thanks for the PR @Cosaquee! We should really use a markdown validator...

**Comment by Cosaquee** [comment]
@bastelfreak do you have any specific validator in mind ? I found https://github.com/mivok/markdownlint and I can probably spend some time and see how it works and connect it to this repo, in form of rake task. : )

**Comment by bastelfreak** [comment]
We manage all our rake tasks with a central modulesync_config repository. We already have an (older) issue about markdown linting. https://github.com/voxpupuli/modulesync_config/issues/182

Contributions are always welcome. Feel free to comment on it or provide an example. You can add a linter to any of our repos and we can later adopt it to modulesync, which then provides it to all of our modules.

**Comment by Cosaquee** [comment]
For now i think it is good idea to close this PR so README can be readable again. In mean time i will research problem of Markdown linting and open new PR if i will have some code ready.

**Comment by bastelfreak** [comment]
please check my inline comment. I think we should switch limitations to Limitations? Since all the other headlines start with a capital letter.

**Comment by Cosaquee** [comment]
@bastelfreak done :)

**Comment by bastelfreak** [comment]
can you check the readme again? There are: '###Usage zabbix-agent' and '###Usage zabbix-proxy' which also need an additional whitespace.

**Pull request merged**